tests/test_fiod.sh

```diff
@@ -4,35 +4,33 @@ set -xeEo pipefail
 source tests/common.sh
 
 function finish {
-  if [ $? -eq 1 ] && [ $ERRORED != "true" ]
-  then
-    error
-  fi
-
-  echo "Cleaning up Fio"
-  kubectl delete -f tests/test_crs/valid_fiod.yaml
-  delete_operator
+  kubectl delete all --all -n my-ripsaw
 }
 
 trap error ERR
 trap finish EXIT
 
 function functional_test_fio {
-  figlet $(basename $0)
+  wait_clean
   apply_operator
-  kubectl apply -f tests/test_crs/valid_fiod.yaml
+  test_name=$1
+  cr=$2
+  echo "Performing: ${test_name}"
+  kubectl apply -f ${cr}
   uuid=$(get_uuid 20)
-  
   wait_for_backpack $uuid
-
   pod_count "app=fio-benchmark-$uuid" 2 300
   fio_pod=$(get_pod "app=fiod-client-$uuid" 300)
-  wait_for "kubectl wait --for=condition=Initialized pods/$fio_pod --namespace my-ripsaw --timeout=200s" "200s" $fio_pod
-  wait_for "kubectl wait --for=condition=complete -l app=fiod-client-$uuid jobs --namespace my-ripsaw --timeout=500s" "500s" $fio_pod
-  sleep 30
+  wait_for "kubectl wait --for=condition=Initialized pods/$fio_pod -n my-ripsaw --timeout=200s" "200s" $fio_pod
+  wait_for "kubectl wait --for=condition=complete -l app=fiod-client-$uuid jobs -n my-ripsaw --timeout=500s" "500s" $fio_pod
   # ensuring the run has actually happened
-  kubectl logs "$fio_pod" --namespace my-ripsaw | grep "fio has successfully finished sample"
-  echo "Fio distributed test: Success"
+  kubectl logs "$fio_pod" -n my-ripsaw
+  kubectl logs "$fio_pod" -n my-ripsaw | grep "fio has successfully finished sample"
+  echo "${test_name} test: Success"
+  kubectl delete -f ${cr}
```

Comment:
This is redundant with wait_clean

Reply:
wait_clean does not remove Benchmark CRs